### PR TITLE
Arch linux fixes

### PR DESCRIPTION
**Pull request by weiss-d**
Finally working successfully  with Arch-based distros (tested on Arco Linux and Hash Linux).
See commit comments at the bottom of the page for details.
Need to re-merge #17 though for everything to work properly (see my comment there).

### CODE REVIEW

File: setup [hunk 1]
```diff
@@ -185,7 +200,7 @@ superbbootstrap_install_packages() {
 		eval "install_command=\"\$${packagemanager}_COMMAND\""
 
 		# Find packages with mark
-		packages_installing=$(printf '%s' "$packages_list" | sed -e "s/^.*$mark://g" -e 's/[[:space:]].*$//g' -e '/^$/d' | tr '\n' ' ')
+		packages_installing=$(printf '%s' "$packages_list" | sed -rn "s/$mark:([0-9a-zA-Z-]+)(.*)/\1/p"  | tr '\n' ' ')
```
Comment (NNBnh) [inline setup:203]: @weiss-d Nice, but `sed -r` is not `POSIX`-compatible thus make the script not portable to run on `BSD` machine.

```
SED(1P)               POSIX Programmer's Manual               SED(1P)

PROLOG
       This  manual  page  is  part of the POSIX Programmer's Manual.
       The Linux implementation of this interface may differ (consult
       the  corresponding  Linux manual page for details of Linux be‐
       havior), or the interface may not be implemented on Linux.

NAME
       sed — stream editor

SYNOPSIS
       sed [-n] script [file...]

       sed [-n] -e script [-e script]... [-f script_file]... [file...]

       sed [-n] [-e script]... -f script_file [-f script_file]... [file...]
```

###### No `-r` flag found...

Comment (weiss-d) [reply setup:204]: Well, I see.
But does the initial command work correctly on BSDs? If yes, we could just make an `if` switch here.

Comment (LoricAndre) [reply setup:204]: You could do this by chaining a `d` sed command in there. I'll try to take a look, see if I can optimize all this.

Comment (LoricAndre) [reply setup:204]: I ended up with this :
```bash
packages_installing=$(printf '%s' "$packages_list" | sed -e "/^.*$mark:.*\$/!d;s/.*$mark://g" -e 's/[[:space:]].*$//g;/^$/d' | tr '\n' ' ')

```

and you can tweak slightly the way comments are stripped with `packages_list=$(sed -e 's/#.*$//g;/^$/d' "$1")`, this strips empty lines directly so they don't have to be processed in each iteration.

Comment (weiss-d) [reply setup]: @LoricAndre big thanks! Added your variant in  7b18550. Tested in Arch.

**Pull request merged**